### ISSUE TITLE
Spellchecking highlights.

### ISSUE THREAD

**Issue by artelse** [rougier/nano-theme#15]
Spellchecking now highlights the whole word, to me this is a bit too emphasized. Are there plans to make these less intrusive; i.e. underline or similar? (And no, no bright red crinkles ;)
[via Emacsconf2021]

**Comment by rougier** [comment]
I get the same feeling actually. Maybe a simple underline in salient color would work. Do you want to make a PR?

**Comment by artelse** [comment]
I've experimented with underlining:
```
(custom-set-faces
   `(flyspell-incorrect ((t (:underline (:color ,nano-light-salient :style line)))))
   `(flyspell-duplicate ((t (:underline (:color ,nano-light-salient :style line))))))
```
which just puts an underline on the words. But am not that pleased with how emacs applies the underline. It is almost on the baseline of the font which is not adjustable as far as I can see. Another option might be to add a faint background or tune the color intensity of the font down, but that's quite non-standard.

**Comment by artelse** [comment]
It appears one can control the underline offset. Here is my current configuration for flyspell and highlighting. Note that I am not familiar yet with github pr's, but is on my todo list to learn about.
``` 
(custom-set-faces
   `(flyspell-incorrect ((t (:underline (:color ,nano-light-salient :style line) :foreground "default" ))))
   `(flyspell-duplicate ((t (:underline (:color ,nano-light-salient :style line) :foreground "default" )))))
(setq underline-minimum-offset 3)
```

**Comment by rougier** [comment]
Thanks. For the PR, you can do directly from GitHub by first forking the repository and edit the file onine. Then GitHub will propose you to make a PR automatically.

For the line offset, does it change the line spacing with a value of 3?

**Comment by artelse** [comment]
Yes, it does. Just saw that [Emacs 29](https://gitlab.com/ganserla/emacs/-/commit/4f50d964e51bbe5219f40df4353f4314c7ade985) enables a `:position` property on `:underline`. 
Regarding the PR, will do, but am severely short on time atm.

**Comment by rougier** [comment]
No problem, I can wait (but I won't be able to test, I'm still using emacs 27)

**Comment by artelse** [comment]
Ok, am still learning about Emacs and currently in the process of setting up with git and local editing of repos, etc.

**Comment by rougier** [comment]
I've uploaded a few changes to the theme including flyspell which is a bit lighter:

<img width="503" alt="Screenshot 2022-01-31 at 17 40 58" src="https://user-images.githubusercontent.com/327203/151835190-3c1c2037-f64b-43c7-b473-3dbffe8d6186.png">
